etc/mkdocsjs/searchhack.js

```diff
@@ -0,0 +1,29 @@
+// Please see https://github.com/commercialhaskell/stack/issues/3376
+
+(function () {
```

Comment:
Haven't tested it, but it looks to me like this function won't be run, needs another () on the last line.